spark-2.0-on-suse/azuredeploy.json

```diff
@@ -0,0 +1,416 @@
+{
+  "$schema": "http://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "adminUserName": {
+      "type": "securestring",
+      "metadata": {
+        "description": "The Linux VM user name"
+      }
+    },
+    "adminPassword": { "type": "securestring",
+      "metadata": {
+        "description": "The Linux VM password"
+      }
+    },
+	"numberOfSlaves" : {
```

Comment:
just remove this altogether and use the default value for the variable you have below...